### ISSUE TITLE
Some proxy objects shouldn't be proxies

### ISSUE THREAD

**Issue by skurfer** [quicksilver/Quicksilver#654]
For instance, see #217. It occurred to me that if “Mounted Disks” was just an object instead of a proxy object, you could get to the children with a single right-arrow. For an example, see the AirPort object added to the catalog by that plug-in.

It generates children on-the-fly only when arrowed into. Another example can be found in the [unreleased Safari plug-in](/quicksilver/com.apple.Safari-qsplugin/pull/1) (Open Web Pages).

I’m not sure what to call these types of objects. Proxy object seems appropriate, but they aren’t literally proxy objects from Quicksilver’s point of view. Some people might not like the idea of mixing these in with all the regular objects. I’m not opposed to somehow identifying them as special in the catalog prefs (maybe redefining the proxy objects category to include things of this type), but none of that is necessary to actually implement these things.

So, in other words, Mounted Disks (and probably a few others) should be redefined not to be proxy objects.
